__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const { app } = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data/index.js");
+
+afterAll(() => {
+    return db.end()
+});
+beforeEach(() => {
+    return seed(data)
+});
+
+describe("get /api/categories", () => {
+    test("responds with JSON containing an array of category objects with a key of 'categories'", () => {
+        return request(app)
+            .get("/api/categories")
+            .expect(200)
+            .then((res) => {
+                expect(res.body).toEqual({
+                    categories: [

```

Comment:
Test could be more generic, use jest matchers to outline the object keys and expected value data types and can then use a length property to check number of entries returned. Have a look at toMatchObject